src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1250,7 +1250,7 @@ export default {
   inherits: true;
 }
 
-.content {
```

Comment:
If you use root than themening can not applied here!
Theming is only available from body upwards

Reply:
🙈

Reply:
Had to learn this when developing *forms* and settings styles on root, but on root only the default theme variables are available.
If you have styles from other themes they are set on body, so the variables are not defined :pensive: